utils/src/unit-test/java/uk/gov/moj/sdt/utils/AbstractSdtUnitTestBase.java

```diff
@@ -138,7 +110,9 @@ public Object getAccessibleField(final Class<?> clazzUnderTest, final String fie
         if (null == field) {
             field = ReflectionUtils.findField(clazzUnderTest, fieldName, clazzOfField);
         }
-        ReflectionUtils.makeAccessible(field);
+        if (null != field) {
+            ReflectionUtils.makeAccessible(field);
+        }
         return ReflectionUtils.getField(field, target);
```

Comment:
I can see that adding the extra if statement gives some additional protection against a null object error, but field could still be null when it reaches the return statement and throw a null object error then.  Perhaps add a final null check and throw a runtime exception if it's still null, e.g.

if (null = field) {
    field = ReflectionUtils.findField(...);
}
if (null = field) {
    // Still can't find the field
    throw new RuntimeException("Field " + fieldName + " does not exist");
}